### DISCUSSION TITLE
building in ubuntu 22.04, missing gtkmm-4.0

### DISCUSSION THREAD

**Discussion by wsteffe** [dune3d/dune3d#78] [Q&A]
Unfornunately gtkmm-4.0, which seems to be a dune3d dependency, isn't available in ubuntu 22.04 (the current LTS). 
I have tried with libgtkmm-3.0-dev but it is not accepted by meson. Have I to wait the next LTS (ubuntu 24.04) ?

**Comment by carrotIndustries** [comment]
> Have I to wait the next LTS (ubuntu 24.04) ?

If you want to stay on LTS, yes.
